content/news/buzz/april-2024-status-update/contents.lr

```diff
@@ -0,0 +1,56 @@
+title: April 2024 Status Update
+---
+author: Russell Keith-Magee
+---
+body:
+
+This month has seen BeeWare reach one major milestone, almost reach a second, as well as landing several smaller improvements.
+
+What we've done
+----------------
+
+* iOS is `officially a Tier 3 supported platform for CPython <https://peps.python.org/pep-0011/#tier-3>`__! All the patches needed to support iOS have been merged, and a `buildbot has been deployed <https://buildbot.python.org/all/#/builders/1380>`__ to verify that CPython can run on iOS. As a result, `PEP 730 has been moved into "Final" status <https://peps.python.org/pep-0730/>`__.
+
+* Android support in CPython is getting very close to completion. The last three patches (adding a `test harness for running the test suite on Android <https://github.com/python/cpython/pull/117878>`__, `redirecting stdout and stderr to the Android log <https://github.com/python/cpython/pull/118063>`__, and `updating sysconfig to include an Android <https://github.com/python/cpython/pull/118352>`__) are currently in review; once those have been merged, and a buildbot has been deployed, Android should also be at Tier 3 status.
+
+* We landed a new `Geolocation service for Toga <https://github.com/beeware/toga/pull/2462>`__, providing access to GPS data on mobile devices and macOS.
+
+* We made some `modifications to how Toga handles application icons <https://github.com/beeware/toga/pull/2527>`__, removing the need to duplicate the icon in application data. These changes are in final review.
+
+* We added a new wizard to `add a Briefcase configuration to an existing Python project <https://github.com/beeware/briefcase/pull/1231>`__.
+
+* We `modified how Briefcase handles splash screens on mobile devices <https://github.com/beeware/briefcase/pull/1751>`__, and corrected some issues with icons seen on recent Android releases.
+
+* We `corrected a problem in Rubicon ObjC <https://github.com/beeware/rubicon-objc/pull/446>`__ caused by a change to the `ctypes` module in CPython 3.13. It might be possible to upstream this fix to CPython itself; but for now, Rubicon ObjC is able to support Python 3.13.0a6.
```

Comment:
```suggestion
* We `corrected a problem in Rubicon ObjC <https://github.com/beeware/rubicon-objc/pull/446>`__ caused by a change to the ``ctypes`` module in CPython 3.13. It might be possible to upstream this fix to CPython itself; but for now, Rubicon ObjC is able to support Python 3.13.0a6.
```